app/controllers/account_controller.rb

```diff
@@ -1,11 +1,11 @@
-class ProfileController < ApplicationController
+class AccountController < ApplicationController
   before_action :confirm_two_factor_authenticated
   layout 'card_wide'
 
   def index
```

Comment:
if we called this `AccountsController`, we'd probably want this to be `show` rather than `index`